### ISSUE TITLE
DTLS support

### ISSUE THREAD

**Issue by fbexiga** [zmap/zgrab2#217]
Any chance that support for DTLS (TLS over UDP) could be added?

**Comment by fbexiga** [comment]
Is there any development on this? I'd be happy to help with examples of live servers and such.

**Comment by phillip-stephens** [comment]
Wondering what would be the best use case for this. Either:

1. DTLS becomes a module. We can then detect hosts that support DTLS and return info similar to the `TLS` module.
2. DTLS is the transport for other modules. We'd add some flag like `--use-dtls` and whatever application-handshake would happen on-top of DTLS.

Off-hand, I'd imagine the use-case here would be for #1, to detect hosts that support DTLS and print out TLS-module-like info on them. 

For reference, it seems that Pion's DTLS implementation is the only major one in the Go ecosystem, and they currently only have support for DTLS 1.2. (Though there's an open [issue](https://github.com/pion/dtls/issues/188) with recent activity discussing adding support for 1.3).

**Comment by fbexiga** [comment]
Yes the use case would be #1. Does the fact that this thread is finally alive means there's hope?

**Comment by phillip-stephens** [comment]
It's not in my roadmap for things I'll personally work on for the next couple months unfortunately.
